### DISCUSSION TITLE
fbw Installer: suggestion to add 3 different folders for experimental, development, stable version

### DISCUSSION THREAD

**Discussion by PatriceCesson** [flybywiresim/aircraft#4490] [Ideas]
Hi,
Could it be added to the fbw installer the possibility to define 3 different folders for installation of stable, experimental and developement versions?
This could be added by defining 3 paths in the settings, each one  related to each version.
Thanks.
Patrice.

**Comment by tracernz** [comment]
Duplicate of https://github.com/flybywiresim/installer/issues/194
